routes/truckRoutes.js

```diff
@@ -1 +1,38 @@
-// put your Express router code in here
\ No newline at end of file
+var express = require('express');
+var trucks = require('../trucks');
+
+var router = express.Router();
+
+router.route('/')
+.get(function(request, response){
+  var truckList = trucks.getTrucks();
+  response.send(truckList);
+})
+.post(function(request, response){
+  var newTruck = trucks.addTruck(); 
```

Comment:
In your POST route, you need to get the request.body first, as this is what will contain the properties for the new truck. The addTruck method takes a truck object - since none was passed in, it will ultimately end up adding undefined.